code/modules/admin/ToRban.dm

```diff
@@ -38,9 +38,9 @@
 			F["last_update"] << world.realtime
 			log_misc("ToR data updated!")
 			if(usr)	usr << "ToRban updated."
-			return 1
+			return

```

Comment:
But why though? If I ran this proc manually I'd want to know if it succeeded or not...

Reply:
spacemandmm says that using a spawn messes up the result either way, and in any case, there's a `usr << "ToRban updated."` there